CHANGELOG.next.md

```diff
@@ -22,6 +22,7 @@ Thanks, you're awesome :-) -->
 * Added `threat.technique.subtechnique` to capture MITRE ATT&CK® subtechniques. #951
 * Added `configuration` as an allowed `event.category`. #963
 * Added network directions ingress and egress. #945
+* Added SIP RFC #tbd
```

Comment:
Don't worry about adding CHANGELOG entries for RFC PRs.

Once the RFC has advanced to a later stage, the PR which adds the field definitions to the schema can be listed.

Reply:
Indeed, please remove the changelog line, @dainperkins . No need for those in RFCs.